route/index.js

```diff
@@ -97,21 +97,33 @@ client
  *
  * @param data {object} route specification
  */
-const drawRoutePolyline = data => {
-  const decodedData = mapboxPolyline.decode(data.polyline);
+const drawRoutePolyline = (route, alternatives) => {
```

Comment:
The function names says drawRoutePolyline though that's not what this function does. Can you think of a better name?